Editor/AGXUnityEditor/Menus/TopMenu.cs

```diff
@@ -385,6 +386,20 @@ public static GameObject Simulation()
       return Selection.activeGameObject = GetOrCreateUniqueGameObject<Simulation>()?.gameObject;
     }
 
+    [MenuItem( "AGXUnity/Plot", priority = 66 )]
+    public static GameObject Plot()
+    {
+      var PlotObject = new GameObject("PlotObject");
+      PlotObject.AddComponent<AGXUnity.Utils.Plot>();
+      PlotObject.AddComponent<AGXUnity.Utils.DataSeries>();
+      PlotObject.AddComponent<AGXUnity.Utils.DataSeries>();
+      var plotAssetPath = AGXUnityEditor.IO.Utils.AGXUnityResourceDirectory + "/Plot/Plot.Asset";
+      if (!System.IO.File.Exists("Assets/Plot.Asset"))
+        AssetDatabase.CopyAsset(plotAssetPath, "Assets/Plot.Asset");
+
+      return Selection.activeGameObject = PlotObject.gameObject;
```

Comment:
Should a ScriptMachine be added to this plot object as well when we create it?

Reply:
I wanted to do this, however it does not seem possible? I can not find Script Machine when I want to add a new component.